internal/pipe/build/build_test.go

```diff
@@ -79,7 +79,11 @@ func TestBuild(t *testing.T) {
 		Version: "1.2.3",
 		Config:  config,
 	}
-	error := doBuild(ctx, ctx.Config.Builds[0], "darwin_amd64")
+	opts, err := buildOptionsForTarget(ctx, ctx.Config.Builds[0], "darwin_amd64")
+	if err != nil {
```

Comment:
require.NoError could be used from testify lib.
https://pkg.go.dev/github.com/stretchr/testify/require?tab=doc#NoError